Makefile-os

```diff
@@ -11,17 +11,17 @@ help_submake:
 
 .PHONY: update_docker
 update_docker: ## update all the docker images
-	docker-compose exec worker make update_deps
```

Comment:
Why the change from `exec` to `run` here and below ? The container should already be running, we don't want another one, we want to re-use the one we have.

Reply:
As written in the description - I noticed that when using `exec` the entrypoint would simply not be executed which lead to many of the permission problems we've seen in folders on the hosts system.

`run` on the other hand does run the entrypoint properly. I haven't found another reasonable solution to this and I'm perfectly aware that it's not ideal but as the original `web` container stays up I don't think it's an issue.

Also, the storage layers (`/deps/`) seem to be shared between those two so that updates do happen on both afaik.

Happy to use `exec` if you know any way to make the entrypoint being executed properly.